server.js

```diff
@@ -14,6 +14,8 @@ app.use('/', expressStaticGzip(`${__dirname}/build`, {
 	index           : false
 }));
 
+process.chdir(__dirname);
```

Comment:
@G-Ambatte, could you please comment about why this line is needed?

Reply:
This has been answered elsewhere, but for historical completeness, I'll include the answer here:

> node.js by default assumes any directories/files you name are relative to the calling directory. __dirname is the directory of the current running file (so the root folder, since that's where server.js resides). Normally, we execute server.js from the root, but in the case of running it from a docker container or BSD jail you might actually start up the app from some external source, which means the app would not be able to find any of the relative files. process.chdir(__dirname); ensures that the "working directory" of the app is always the same root folder no matter from where it is started up.